Dwolla.Client/Rest/RestResponse.cs

```diff
@@ -6,13 +6,15 @@ public class RestResponse<T>
     {
         public HttpResponseMessage Response { get; }
         public T Content { get; }
+        public string RawContent { get; }
         public RestException Exception { get; }
 
-        public RestResponse(HttpResponseMessage response, T content, RestException exception = null)
+        public RestResponse(HttpResponseMessage response, T content, RestException exception = null, string rawContent = null)
```

Comment:
Since `rawContent` is always passed, can we move it before the exception parameter and require it (so don't set to null)? Also, tests should be updated to check that `RawContent` is getting set.

Reply:
I was trying to avoid changing the constructor signature, but I can do that.